src/WCSimRunAction.cc

```diff
@@ -155,14 +155,14 @@ void WCSimRunAction::BeginOfRunAction(const G4Run* /*aRun*/)
   flatfile->SetCompressionLevel(2); //default is 1 (minimal compression)
   masterTree = new TTree("MasterTree","Main WCSim Tree");
   if(wcsimdetector->GetIsNuPrism()){
-    if(GetSaveRooTracker())
+    if(GetSaveRooTracker()){
```

Comment:
I think you need to add another "else geomTree = new TTree("Geometry","Geometry Tree");" line after the closing brace on line 165